src/elements/content-uploader/resumeCellRenderer.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @flow
+ * @file Function to render the resume table cell
+ */
+
+import React from 'react';
+import ItemResume from './ItemResume';
+
+type Props = {
+    rowData: UploadItem,
+};
+
+export default (onClick: Function) => ({ rowData }: Props) => (
+    <ItemResume {...rowData} onClick={() => onClick(rowData)} />
```

Comment:
Minor suggestion: Should pass a reference to a function as to not create a new function on each render.